### PR TITLE
[ASPA-12] Add home button

### PR DESCRIPTION
**Pull request by 3Bies**
Issue:
Aspa form submitted page requires a home button to take the user back to the main page.

Solution:
By creating a home button and using an anchor to link to the main page.

Risk:
None

Reviewed By:
[Edit the message as to who reviewed it here]

### CODE REVIEW

File: application/views/PaymentSuccessful.php [hunk 1]
```diff
@@ -20,9 +20,15 @@
     </head>
 
     <body>
+        
+
```
Comment (Ray-F) [inline application/views/PaymentSuccessful.php]: Remove these spaces

File: application/views/PaymentSuccessful.php [hunk 2]
```diff
@@ -20,9 +20,15 @@
...
         <div class="centre-page">
+            <!--
+            The href of this button link is left empty as it would otherwise not work on localhost.
+            When the page is finished, this should be changed to the websites url "aspa.wdcc.co.nz"
+            -->
```
Comment (Ray-F) [inline application/views/PaymentSuccessful.php]: This comment is no longer needed now that `...href="/"...` has been updated

File: application/views/PaymentSuccessful.php [hunk 3]
```diff
@@ -20,9 +20,15 @@
...
+            <br><a href="/"><img align="left" alt="Home Page Button" alt="Home Page" src="assets/images/iconmonstr-home-grey.svg" width="30px"></a><br><br>
```
Comment (Ray-F) [inline application/views/PaymentSuccessful.php]: 1) Can you close the `<br />` statements and put these on separate lines for greater code visibility? Also, close the `img` tag too.

2) Let's also separate out the `<a>` tag and the `` tag on separate lines and indented, like so...
```
<br />
<a ...>
    <img ... />
</a>
<br />
```

**Review by Ray-F — CHANGES_REQUESTED** [review]
Overall, looks really good and behaves when I tested it on my local machine!

Just some really minor changes I suggested to improve the code readability/visibility. Sorry if they come across as too serious or strong toned, but it's just for clarity of what I mean.


Good work! 👍

**Review by mouyang2001 — APPROVED** [review]
Looks good! Just implement those small changes as mentioned by Raymond.

**Review by mouyang2001 — COMMENTED** [review]
Looks good to me!

**Pull request merged**